pkg/api-server/api/v1/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
please rename `api-server` folder to `apiserver` as we discussed

Reply:
done